main/app/attentionRelated/ears/conf/client.ini

```diff
@@ -2,3 +2,4 @@ subsystems          (recog)
 reachingRangeMin    (-0.5 -0.3 -0.15)
 reachingRangeMax    (-0.1 0.3 0.5)
 robot               icub
+zOffset             0.0
```

Comment:
Hi @towardthesea, my idea is to use the z-coordinate of the object which is stored in the OPC, and then apply an offset to it (if needed). This is independent of the table height in the various labs, as the object height implicitly tells you the table height.
Furthermore, we can just tell people to calibrate the table height using ARE (if not done already), so this backup case should be rarely triggered and is not worth changing dozens of config files.